server/src/main/java/io/spine/server/procman/ProcessManagerRepository.java

```diff
@@ -100,36 +103,70 @@ protected ProcessManagerRepository() {
 
-        final DelegatingCommandDispatcher<I> commandDispatcher =
-                DelegatingCommandDispatcher.of(this);
+        final BoundedContext boundedContext = getBoundedContext();
+        final DelegatingRejectionDispatcher<I> rejDispatcher =
+                DelegatingRejectionDispatcher.of(this);
 
-        if (!commandDispatcher.getMessageClasses()
-                              .isEmpty()) {
-            boundedContext.getCommandBus()
-                          .register(commandDispatcher);
+        final boolean handlesCommands = register(boundedContext.getCommandBus(),
+                                                 DelegatingCommandDispatcher.of(this));
+        final boolean handlesDomesticRejections = register(boundedContext.getRejectionBus(),
+                                                           rejDispatcher);
+        final boolean handlesExternalRejections = register(boundedContext.getIntegrationBus(),
+                                                           rejDispatcher.getExternalDispatcher());
+        final boolean handlesDomesticEvents = !getMessageClasses().isEmpty();
+        final boolean handlesExternalEvents = !getExternalEventDispatcher().getMessageClasses()
+                                                                          .isEmpty();
+
+        final boolean subscribesToEvents = handlesDomesticEvents || handlesExternalEvents;
+        final boolean reactsUponRejections = handlesDomesticRejections || handlesExternalRejections;
+
+        if (!handlesCommands && !subscribesToEvents && !reactsUponRejections) {
+            throw newIllegalStateException(
+                    "Process managers of the repository %s have no command handlers, " +
+                            "and do not react upon any rejections or events.", this);
         }
 
-        final DelegatingRejectionDispatcher<I> rejectionDispatcher =
-                DelegatingRejectionDispatcher.of(this);
-        if (!rejectionDispatcher.getMessageClasses()
-                                .isEmpty()) {
-            boundedContext.getRejectionBus()
-                          .register(rejectionDispatcher);
-        }
+    }
 
-        final ExternalMessageDispatcher<I> extRejectionDispatcher =
-                rejectionDispatcher.getExternalDispatcher();
-        if (!extRejectionDispatcher.getMessageClasses()
-                                   .isEmpty()) {
-            boundedContext.getIntegrationBus()
-                          .register(extRejectionDispatcher);
+    /**
+     * Registers the given dispatcher in the bus if there is at least one message class declared
+     * for dispatching by the dispatcher.
+     *
+     * @param bus        the bus to register dispatchers in
+     * @param dispatcher the dispatcher to register
+     * @param <D>        the type of dispatcher
+     * @return
+     * {@code true} if there are message classes to dispatch by the given dispatchers,
```

Comment:
This alignment appears strange.

Reply:
Adjusted. My goal was to have `{@code true}` and `{@code false}` inter-aligned.